src/main/java/org/fundacion/jala/converter/controller/AuthController.java

```diff
@@ -20,13 +22,11 @@
 import org.springframework.security.authentication.BadCredentialsException;
 import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
 import org.springframework.security.core.userdetails.UserDetails;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PostMapping;
 
-import static org.fundacion.jala.converter.models.UserSQL.editUserData;
-import static org.fundacion.jala.converter.models.UserSQL.findUserById;
+import static org.fundacion.jala.converter.models.UserSQL.editToken;
 
```

Comment:
javadoc of the class is missing

Reply:
None of the project classes have javadoc yet. A person would be assigned for this task.